src/steps/resource-manager/converters.ts

```diff
@@ -80,6 +82,27 @@ export function createGoogleWorkspaceEntityTypeAssignedIamRoleMappedRelationship
   });
 }
 
+export function createAuditConfigEntity(
+  data: cloudresourcemanager_v3.Schema$AuditConfig,
+) {
+  return createGoogleCloudIntegrationEntity(data, {
+    entityData: {
+      source: data,
+      assign: {
+        _key: `auditConfig:${data.service}`,
```

Comment:
Hmm do we need to use the [new pattern](https://github.com/JupiterOne/graph-google-cloud/pull/275#discussion_r680974932) for the keys here as well (the full URI path) or should this suffice (e.g. this type of resource is different/special and we don't need to use that pattern)?

Reply:
I also thought about this at first but I didn't follow the pattern because I don't think this is a resource supported by the Cloud Asset API which they use for that relationship i.e. we can't build the `_key` for the relationship mapping with an IAM policy because we can't make an IAM binding with this entity as this is not a Google Cloud 'resource'.